### PR TITLE
🌐: Turkish translations

### PR DESCRIPTION
**Pull request by eniyiweb**
# Pull Request Template

## Summary

Please provide a brief summary of your changes and the related issue. Include any motivation and context that is relevant to your changes. If there are any dependencies necessary for your changes, please list them here.

## Change Type

- [x] Translation support

## Testing

Tested on Docker

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented in any complex areas of my code
- [ ] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [ ] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [ ] Any changes dependent on mine have been merged and published in downstream modules.

### CODE REVIEW

File: client/src/localization/languages/Tr.tsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  com_nav_clear_conversation: 'Sohbetleri Temizle',
+  com_nav_clear_conversation_confirm_message:
+    'Tüm sohbetleri temizlemek istediğinizden emin misiniz? Bu geri alınamaz.',
+  com_nav_help_faq: 'Yardım ve SSS',
+  com_nav_settings: 'Ayarlar',
+  com_nav_search_placeholder: 'Mesajları ara',
+  com_nav_setting_general: 'Genel',
+  com_nav_setting_data: 'Veri kontrolleri',
+  com_nav_language: 'Dil',
+  com_nav_lang_auto: 'Otomatik algıla',
```
Comment (berry-13) [inline client/src/localization/languages/Tr.tsx]: remove these 2 translations

**Review by berry-13 — CHANGES_REQUESTED** [review]
Great PR!

File: client/src/localization/languages/Tr.tsx (hunk 1 above)
Comment (eniyiweb) [reply client/src/localization/languages/Tr.tsx]: Removed.

**Review by berry-13 — APPROVED** [review]
Thank you!

**Pull request merged**